sdcm/remote/remote_file.py

```diff
@@ -23,12 +23,29 @@
 
 
 LOGGER = logging.getLogger(__name__)
+# The below set of SECURED_PARAMETERS should include relevant password-parameter-names,
+# to be masked by asterisks when printed to log.
+SECURED_PARAMETERS = {'ldap_bind_passwd'}
 
 
 def read_to_stringio(fobj):
     return StringIO(fobj.read())
 
 
+def secured_masked_content(parameters_content: str) -> str:
+    yaml_content = {}
+    out = yaml.safe_load(parameters_content)
```

Comment:
it's placed under `remote_file` - which is not always yaml and content may cause it to raise exception and do actual harm.

Reply:
`yaml` is yet another valid type of a remote file. Please let me know if there's any better suggestion.
any line of code in python may raise an exception.. that's how python is built.